### DISCUSSION TITLE
Hexarotor with SITL

### DISCUSSION THREAD

**Discussion by manthan99** [microsoft/AirSim#4553] [Support Q&A]
Is it possible to run the hexarotor or any other frame in Software in Loop mode(PX4) with Airsim? I was unable to find any material related to this.

[https://github.com/microsoft/AirSim/wiki/hexacopter](https://github.com/microsoft/AirSim/wiki/hexacopter)
#941 
#2424 
https://github.com/microsoft/AirSim/pull/1890

All the above-referenced issues deal with HIL

**Comment by lovettchris** [comment]
Yes it is possible in SITL, you just need to ensure when you launch px4 it ends up using a hex rotor mixer from \px4\Firmware\ROMFS\px4fmu_common\mixers\ like hexa_x.main.mix.  Note that `make px4_sitl_default none_iris` picks up this file: ./ROMFS/px4fmu_common/init.d-posix/10016_iris
which is using `set MIXER quad_w`

**Reply by latif11252** [reply]
still could not do it :( , can u please explain more

**Comment by latif11252** [comment]
did you find any solution for this problem, bcz am struggling here :(
